CryptoPkg/Library/BaseCryptLib/SysCall/CrtWrapper.c

```diff
@@ -580,15 +582,168 @@ fopen (
 
 size_t
 fread (
-  void    *b,
-  size_t  c,
-  size_t  i,
-  FILE    *f
+  void    *buffer,
+  size_t  size,
+  size_t  count,
+  FILE    *stream
   )
 {
   return 0;
 }
 
+int
+fputs (
+  const char  *str,
+  FILE        *stream
+  )
+{
+  return -1;
+}
+
+int
+fflush (
+  FILE  *stream
+  )
+{
+  return -1;
+}
+
+int
+ferror (
+  FILE  *fp
+  )
+{
+  return -1;
+}
+
+int
+fseek (
+  FILE  *stream,
+  long  offset,
+  int   whence
+  )
+{
+  return -1;
+}
+
+int
+feof (
+  FILE  *fp
+  )
+{
+  return -1;
+}
+
+int
+ftell (
+  FILE  *fp
+  )
+{
+  return -1;
+}
+
+char *
+fgets (
+  char  *str,
+  int   n,
+  FILE  *stream
+  )
+{
+  return NULL;
+}
+
+char *
+strdup (
+  char  *strSource
+  )
+{
+  UINTN  len;

```

Comment:
```suggestion
  UINTN  Length;
```
Follow EDK2 style guide.